src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -46,6 +47,18 @@
 from fairseq2.typing import DataType, Device
 
 
+@dataclass
+class ShawRelativePositionSDPAConfig:
```

Comment:
Nit: Maybe add a brief docstr like: "Holds the configuration of a :class:`ShawRelativePosition` module", so that people can jump to the implementation to learn more about the parameters.